updater/lib/dependabot/updater/group_update_creation.rb

```diff
@@ -18,6 +19,8 @@ module GroupUpdateCreation
       # outcome of attempting to update every dependency iteratively which
       # can be used for PR creation.
       def compile_all_dependency_changes_for(group)
```

Comment:
We now get an overall debug spiel that looks like this for each step:

```
updater | 2023/06/23 13:42:33 DEBUG File deleted: '/bundler-vendored/vendor/cache/rack-2.1.4.gem'
updater | 2023/06/23 13:42:33 DEBUG File created: '/bundler-vendored/vendor/cache/rack-3.0.8.gem'
updater | 2023/06/23 13:42:33 DEBUG Dependencies updated:
updater | 2023/06/23 13:42:33 DEBUG  - rubocop ( 0.76.0 to 1.53.0 )
updater | 2023/06/23 13:42:33 DEBUG  - rack ( 2.1.4 to 3.0.8 )
updater | 2023/06/23 13:42:33 DEBUG Dependency files updated:
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/Gemfile ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/Gemfile.lock ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG Vendored dependency changes:
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/jaro_winkler-1.5.4.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/parallel-1.22.1.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/parser-3.1.2.0.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/rubocop-0.76.0.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/ruby-progressbar-1.11.0.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/unicode-display_width-1.6.1.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/json-2.6.3.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/language_server-protocol-3.17.0.3.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/parallel-1.23.0.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/parser-3.2.2.3.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/racc-1.7.1.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/regexp_parser-2.8.1.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/rexml-3.2.5.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/rubocop-1.53.0.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/rubocop-ast-1.29.0.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/ruby-progressbar-1.13.0.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/unicode-display_width-2.4.2.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/rack-2.1.4.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG   - /bundler-vendored/vendor/cache/rack-3.0.8.gem ( Changed 1 times )
updater | 2023/06/23 13:42:33 DEBUG Storing change to workspace: Updating rack
updater | 2023/06/23 13:42:33 DEBUG [workspace] store_change - before: 11b45e5 Updating rubocop
updater | 2023/06/23 13:42:33 DEBUG [workspace] store_change - after: 1118b5c Updating rack
```

Reply:
I've definitely messed up the `change_count` though, I'll look at that separately.